README.md

```diff
@@ -10,6 +10,7 @@ Tools for Clinical and Community Data Initiative (CODI) data owners to extract p
 1. [Extract PII](#extract-pii)
 1. [Garbling](#garbling-pii)
 1. [Mapping LINKIDs to PATIDs](#mapping-linkids-to-patids)
+1. [Verifying Result Metadata](#verify-result-metadata)
```

Comment:
Verifying the metadata & link_ids received from the linkage-agent should happen before actually using those artifacts, no?
```suggestion
1. [Verifying Result Metadata](#verify-result-metadata)
1. [Mapping LINKIDs to PATIDs](#mapping-linkids-to-patids)
```

Reply:
See the newest version of the README, it accurately reflects the rest of the changes